### ISSUE TITLE
--no-cext option on setup.py doesn't seem to work

### ISSUE THREAD

**Issue by eggplantbren** [eggplantbren/DNest4#5]
```
[brendon@brendonlaptop python]$ python setup.py --no-cext install
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: option --no-cext not recognized
```

**Comment by dfm** [comment]
https://github.com/eggplantbren/DNest4/blob/master/python/setup.py#L19

`--without-cext`

**Comment by eggplantbren** [comment]
In that case, the bug is in python/README.rst. I'll fix it.

**Comment by eggplantbren** [comment]
Fixed in commit 97d0df68c37417ea48984d434fc08eb150134836
